CRITICAL - 2021-09-15 05:18:25 --> Call to a member function get_by_date() on null
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:18:51 --> Class 'App\Controllers\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:19:10 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:19:19 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:19:48 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:21:09 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:22:17 --> Call to undefined method App\Models\M_cdms_customer::get_all_cus
#0 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(111): CodeIgniter\Model->__call('get_all_cus', Array)
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-09-15 05:22:35 --> Call to undefined method App\Models\M_cdms_customer::get_all_cus
#0 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(111): CodeIgniter\Model->__call('get_all_cus', Array)
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-09-15 05:22:37 --> Call to undefined method App\Models\M_cdms_customer::get_all_cus
#0 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(111): CodeIgniter\Model->__call('get_all_cus', Array)
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-09-15 05:22:59 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:24:02 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:24:29 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:27:44 --> require(../../vendor/autoload.php): failed to open stream: No such file or directory
#0 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(131): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require(../../v...', 'C:\\xampp\\htdocs...', 131, Array)
#1 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(131): require()
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#4 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-15 05:27:50 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:28:57 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 05:30:09 --> Call to undefined method App\Models\M_cdms_customer::get_all_cus
#0 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(111): CodeIgniter\Model->__call('get_all_cus', Array)
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-09-15 05:31:15 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 07:17:04 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 07:27:20 --> Call to a member function get_all() on null
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 07:27:42 --> array_column() expects parameter 1 to be array, null given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'array_column() ...', 'C:\\xampp\\htdocs...', 184, Array)
#1 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(184): array_column(NULL, 'ser_type')
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#4 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-15 07:29:31 --> array_column() expects parameter 1 to be array, null given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'array_column() ...', 'C:\\xampp\\htdocs...', 184, Array)
#1 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(184): array_column(NULL, 'ser_type')
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#4 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-15 07:52:19 --> count(): Parameter must be an array or an object that implements Countable
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'count(): Parame...', 'C:\\xampp\\htdocs...', 118, Array)
#1 C:\xampp\htdocs\code_team4\app\Controllers\Customer_show.php(118): count(NULL)
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#4 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-09-15 08:40:30 --> Class 'App\Controllers\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Service_show->export_service()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 08:41:28 --> Too few arguments to function PhpOffice\PhpSpreadsheet\Worksheet\Worksheet::setCellValue(), 1 passed in C:\xampp\htdocs\code_team4\app\Controllers\Service_show.php on line 239 and exactly 2 expected
#0 C:\xampp\htdocs\code_team4\app\Controllers\Service_show.php(239): PhpOffice\PhpSpreadsheet\Worksheet\Worksheet->setCellValue('C\xE0\xB8\x99\xE0\xB8\xB3\xE0\xB8\x95\xE0\xB8\xB9\xE0\xB9...')
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Service_show->export_service()
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service_show))
#3 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-09-15 11:03:01 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 11:03:14 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 11:03:38 --> Class 'PhpOffice\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Customer_show->export_customer()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customer_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 23:50:05 --> Class 'ThirdParty\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Service_show->export_service()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 23:50:30 --> Class 'App\ThirdParty\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Service_show->export_service()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2021-09-15 23:51:21 --> Class 'App\ThirdParty\PhpSpreadsheet\Spreadsheet' not found
#0 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(928): App\Controllers\Service_show->export_service()
#1 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(436): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Service_show))
#2 C:\xampp\htdocs\code_team4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\code_team4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
